pkg/cgroups/cgroups_supported.go

```diff
@@ -118,8 +130,16 @@ func rmDirRecursively(path string) error {
 			return nil
 		}
```

Comment:
The file doesn't look `gofmt`ed is it?  The indentation looks off.

Reply:
`gofmt` seems happy about it:

```
$ gofmt -l pkg/cgroups/cgroups_supported.go
$
```